content/post/gke-mariadb/index.md

```diff
@@ -0,0 +1,540 @@
+etcds.kubedb.com                                  2022-01-26T08:36:07Z
+etcdversions.catalog.kubedb.com                   2022-01-26T08:34:30Z
+mariadbopsrequests.ops.kubedb.com                 2022-01-26T08:35:53Z
+mariadbs.kubedb.com                               2022-01-26T08:35:52Z
+mariadbversions.catalog.kubedb.com                2022-01-26T08:34:30Z
+memcacheds.kubedb.com                             2022-01-26T08:36:07Z
+memcachedversions.catalog.kubedb.com              2022-01-26T08:34:30Z
+mongodbautoscalers.autoscaling.kubedb.com         2022-01-26T08:35:10Z
+mongodbopsrequests.ops.kubedb.com                 2022-01-26T08:35:42Z
+mongodbs.kubedb.com                               2022-01-26T08:35:42Z
+mongodbversions.catalog.kubedb.com                2022-01-26T08:34:30Z
+mysqlopsrequests.ops.kubedb.com                   2022-01-26T08:35:49Z
+mysqls.kubedb.com                                 2022-01-26T08:35:49Z
+mysqlversions.catalog.kubedb.com                  2022-01-26T08:34:30Z
+perconaxtradbs.kubedb.com                         2022-01-26T08:36:09Z
+perconaxtradbversions.catalog.kubedb.com          2022-01-26T08:34:30Z
+pgbouncers.kubedb.com                             2022-01-26T08:35:46Z
+pgbouncerversions.catalog.kubedb.com              2022-01-26T08:34:30Z
+postgreses.kubedb.com                             2022-01-26T08:36:00Z
+postgresopsrequests.ops.kubedb.com                2022-01-26T08:36:00Z
+postgresversions.catalog.kubedb.com               2022-01-26T08:34:30Z
+proxysqls.kubedb.com                              2022-01-26T08:36:09Z
+proxysqlversions.catalog.kubedb.com               2022-01-26T08:34:30Z
+redises.kubedb.com                                2022-01-26T08:35:56Z
+redisopsrequests.ops.kubedb.com                   2022-01-26T08:35:56Z
+redissentinels.kubedb.com                         2022-01-26T08:35:56Z
+redisversions.catalog.kubedb.com                  2022-01-26T08:34:30Z
+```
+
+## Deploy Standalone MariaDB Database
+
+Now we are going to Install MariaDB with the help of KubeDB.
+At first, let's create a Namespace in which we will deploy the database.
+
+```bash
+$ kubectl create ns demo
+namespace/demo created
+```
+
+Here is the yaml of the MariaDB CRD we are going to use:
+
+```yaml
+apiVersion: kubedb.com/v1alpha2
+kind: MariaDB
+metadata:
+  name: sample-mariadb
+  namespace: demo
+spec:
+  version: "10.6.4"
+  storageType: Durable
+  storage:
+    storageClassName: "standard"
+    accessModes:
+    - ReadWriteOnce
+    resources:
+      requests:
+        storage: 1Gi
+  terminationPolicy: WipeOut
+```
+
+Let's save this yaml configuration into mariadb.yaml 
+Then create the above MariaDB CRD
+
+```bash
+$ kubectl create -f mariadb.yaml
+mariadb.kubedb.com/sample-mariadb created
+```
+
+* In this yaml we can see in the `spec.version` field the version of MariaDB. You can list the KubeDB supported versions of MariaDB by running `kubectl get mariadbversion` command.
```

Comment:
In this yaml we can see in the `spec.version` field **specifies** the version of MariaDB.